@here/harp-map-controls/lib/MapControls.ts

```diff
@@ -143,6 +143,12 @@ export class MapControls extends THREE.EventDispatcher {
      */
     rotateEnabled = true;
 
+    /**
+     * If defined, restricts all camera movements (pan, zoom, orbit) triggered from these map
+     * controls to the specified bounding box.
+     */
+    bounds?: geoUtils.GeoBox;
```

Comment:
The documentation is a bit imprecise. Do we restrict the target point to be in the bounds or the camera or the whole view frustum?

Reply:
I'll clarify that, it's the camera position.

Reply:
Okay, but maybe we need to clarify with the customer what they actually want. Especially for the usecase where you only want to see "one" world you need to restrict the frustum, not only the camera position.

Reply:
I can double check. But the ticket is quite clear they want to restrict movement, not visibility. 
Quoting https://saeljira.it.here.com/browse/HARP-10717 :
"restrict the camera movements to a particular bounds"

MapBox has also the same functionality.